docs/source/_static/main-page.css

```diff
@@ -44,7 +44,7 @@ body {
 }
 
 .hero {
-  background-color: #000000e0;
```

Comment:
OK, this is interesting.  We actually want this to be slightly transparent, so the `E0` value at the end is important.  Perhaps your browser didn't like the lowercased 'e'?  Does this change fix the situation in your case?

Reply:
We could also use `#202020` instead.  Does that work better in your browser @stefanv ?  This is something that you can probably set manually using your browser's dev tools to see how it looks.  Happy to walk you through this if you're unfamiliar.

Reply:
@mrocklin 
This is invalid color value, please try `background-color: rgba(0,0,0,0.8);`

Reply:
That's the correct solution, I think.

Reply:
(PR updated)